Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:8-alpine
+
+COPY . /app
+WORKDIR /app
+RUN mkdir static
+RUN npm install
+
+EXPOSE 1443
```

Comment:
You should `ENV PORT=1443` and `EXPOSE $PORT`, and the app should accept a `PORT` env variable to listen on. This is because cloudops deploys docker containers with `--net host`, so PORT must be configurable.